src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+
+.stopwatch {
+  position: relative;
+
+  &__border {
+    width: 80vmin;
+    height: 80vmin;
+    border-radius: 50%;
+    border: 1vmin dotted #000;
```

Comment:
The values for `width`, `height`, and `border` are using `vmin` units, which may not be the best choice for responsiveness across different devices. Consider using `em`, `rem`, or percentage units for better scalability and readability.